### PR TITLE
Omit organisation from xAPI responses

### PR DESCRIPTION
**Pull request by francisco-videira-nhs**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: sandbox/src/mocks/patientServiceSearch/responses/FetchServiceListWithMultipleServices.json [hunk 1]
```diff
@@ -91,16 +91,6 @@
                 "url": "referralLetterRequired",
                 "valueBoolean": false
               },
-              {
```
Comment (francisco-videira-nhs) [inline sandbox/src/mocks/patientServiceSearch/responses/FetchServiceListWithMultipleServices.json:94]: Revert

File: specification/components/schemas/extensions/Extension-eRS-ServiceSummaryView.yaml [hunk 2]
```diff
@@ -461,29 +461,6 @@ properties:
             valueBoolean:
               type: boolean
               example: true
-            valueReference:
```
Comment (nhsd-jack-wainwright) [inline specification/components/schemas/extensions/Extension-eRS-ServiceSummaryView.yaml:464]: It looks like this extension is used as part of the response from patient service search so I think this organisation reference will still be returned.

**Review by nhsd-jack-wainwright — COMMENTED** [review]
Looks good to me, just one comment 🙂

File: specification/components/schemas/extensions/Extension-eRS-ServiceSummaryView.yaml (hunk 2 above)
Comment (nhsd-jack-wainwright) [reply specification/components/schemas/extensions/Extension-eRS-ServiceSummaryView.yaml:464]: Ah I remember now, this part of the OAS file is currently incorrect on develop as the `ReferralLetterRequired` extension returned by the API will only be a boolean, not a reference.

**Review by nhsd-jack-wainwright — DISMISSED** [review]
Looks good to me 🙂

**Review by nhsd-jack-wainwright — APPROVED** [review]

**Pull request merged**